Formula/mitmproxy.rb

```diff
@@ -190,8 +189,8 @@ class Mitmproxy < Formula
   end
 
   resource "zstandard" do
```

Comment:
Since we're installing `zstd` bindings, I wonder if this actually has a `zstd` dependency too.